lib/EditableList/processBadResponse.js

```diff
@@ -78,5 +78,12 @@ export default async function processBadResponse(response, defaultErrorCode) {
     }
   }
 
+  if (response.status === 409) {
+    return {
```

Comment:
@AndreiShumski2 could you use `getDefaultErrorMessage('conflict')` here instead?

Reply:
That's a nice idea, and a simpler way to get the same effect. These functions could really use some comments!